packages/remix-dev/cli/migrate/run/index.ts

```diff
@@ -1,24 +1,24 @@
 import { checkGitStatus } from "../../check-git-status";
-import type { transformOptions } from "../transform-options";
+import type { migrationOptions } from "../migration-options";
 import type { Transform, TransformArgs } from "./transforms";
 import { updateRemixImports } from "./transforms";
 import { validateAnswers } from "./validate-answers";
 
 const transformFunctionByName: Record<
-  typeof transformOptions[number]["value"],
+  typeof migrationOptions[number]["value"],
   Transform
 > = {
-  "update-remix-imports": updateRemixImports,
+  "replace-remix-imports": updateRemixImports,
```

Comment:
Probably a good idea to rename `updateRemixImports` to `replaceRemixImports` as well?

Reply:
Right now I just wanted to update the user-facing stuff. Will put up another PR later with some more refactoring to internals including this sort of internal renaming^.